sections/main-product.liquid

```diff
@@ -578,8 +617,9 @@
                           value="{{ variant.id }}"
                         >
                           {{ variant.title }}
-                          {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
                           - {{ variant.price | money | strip_html }}
+                          {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
+                          {% if variant.available and variant.inventory_management == 'shopify' -%} - {{ 'products.product.inventory_in_stock_show_count' | t: quantity: variant.inventory_quantity }}{% endif %}
```

Comment:
Same here as for the feat. product section. We shouldn't show the inventory quantity by default. And we could have an if/elsif rather than 2 if statements 👍